### DISCUSSION TITLE
Custom Player Skulls

### DISCUSSION THREAD

**Discussion by Nopock** [Minestom/Minestom#1271] [Help] [answered]
How can i create a custom player skull with specific textures

**Comment by LooFifteen** [comment, marked as answer]
You'll need to set the skin on the skull using [PlayerHeadMeta.Builder#playerSkin](https://javadoc.minestom.net/net/minestom/server/item/metadata/PlayerHeadMeta.Builder.html#playerSkin(net.minestom.server.entity.PlayerSkin)) and set the UUID to something (I believe it can be random) using [PlayerHeadMeta.Builder#skullOwner](https://javadoc.minestom.net/net/minestom/server/item/metadata/PlayerHeadMeta.Builder.html#skullOwner(java.util.UUID)).

Player skins must be signed my Mojang to be used. If you're trying to use a texture stored locally, I'd recommend using [MineSkin](https://mineskin.org) to convert the images into skin data.

**Reply by Nopock** [reply]
Do you know how I can access PlayerHeadMeta?

**Reply by LooFifteen** [reply]
See #922.
